### ISSUE TITLE
Data-type="html" and sorting by number

### ISSUE THREAD

**Issue by itajackass** [fooplugins/FooTable#462]
i've a

```
<thead>
                <tr>
                  <th data-type="html">Prices</th>
```

and multiple rows like:

```
...
<td data-sorting-value="488.0"><a href="#">488.0</a></td>
...
<td data-sorting-value="274.5"><a href="#">274.5</a></td>
...
<td data-sorting-value="2440.0"><a href="#">2440.0</a></td>
...
```

but sorting isn't correct and i don't know why...

**Comment by itajackass** [comment]
i get a fix using:

```

<?php $value = 488.0; ?>
...
data-sorting-value="<?php echo $value * 100; //to delete 2 digits after "." ?>"
```

**Comment by steveush** [comment]
Hi,

I think you are encountering the same issue as issue #459, please try the fix specified there for now and see if it resolves the issue. I will correct the default value of the `thousandSeparator` option in the next version.

Thanks

**Comment by steveush** [comment]
Hi, this should be fixed in 3.0.10
